scripts/utils.js

```diff
@@ -211,33 +212,54 @@ function traceTx (receipt, cost) {
   }
 }
 
-function traceWitnetPriceFeed (caption, hash, radHash, latestTimestamp) {
+async function traceWitnetPriceFeed (pfs, caption, id4, radHash, latest, liveResult) {
```

Comment:
Making the `trace` function `async` could result in unsorted or mixed logs if `traceWitnetPriceFeed` is called in parallel. Is this an issue in the current context?"